packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -1,13 +1,28 @@
+import * as React from 'react';
 import { styled } from '../../Utilities';
 import { IContextualMenuProps, IContextualMenuStyleProps, IContextualMenuStyles } from './ContextualMenu.types';
 import { ContextualMenuBase } from './ContextualMenu.base';
 import { getStyles } from './ContextualMenu.styles';
 
+// This is to prevent cyclic import with ContextualMenu.base.tsx.
+let LocalContextualMenu: React.StatelessComponent<IContextualMenuProps>;
+
+function onRenderSubMenu(subMenuProps: IContextualMenuProps, _: any) {
```

Comment:
Shouldn't need to specify args if they aren't used. Should be able to remove `_: any` here.

Reply:
These can be removed without further problem.